huaweicloud/services/acceptance/lts/resource_huaweicloud_lts_host_access_test.go

```diff
@@ -0,0 +1,287 @@
+				ImportStateIdFunc: testAccHostAccessConfigImportStateFunc(rName),
+			},
+		},
+	})
+}
+
+func TestAccHostAccessConfig_windows(t *testing.T) {
+	var obj interface{}
+
+	name := acceptance.RandomAccResourceName()
+	rName := "huaweicloud_lts_host_access.test"
+
+	rc := acceptance.InitResourceCheck(
+		rName,
+		&obj,
+		getHostAccessConfigResourceFunc,
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { acceptance.TestAccPreCheck(t) },
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		CheckDestroy:      rc.CheckResourceDestroy(),
+		Steps: []resource.TestStep{
+			{
+				Config: testHostAccessConfig_windows_basic(name),
+				Check: resource.ComposeTestCheckFunc(
+					rc.CheckResourceExists(),
+					resource.TestCheckResourceAttr(rName, "name", name),
+					resource.TestCheckResourceAttr(rName, "access_config.0.paths.0", "D:\\data\\log\\*"),
+					resource.TestCheckResourceAttr(rName, "access_config.0.black_paths.0", "D:\\data\\log\\a.log"),
+					resource.TestCheckResourceAttr(rName, "access_config.0.windows_log_info.0.time_offset", "7"),
+					resource.TestCheckResourceAttr(rName, "access_config.0.windows_log_info.0.time_offset_unit", "day"),
+					resource.TestCheckResourceAttr(rName, "host_group_ids.#", "1"),
+					resource.TestCheckResourceAttr(rName, "access_type", "AGENT"),
+					resource.TestCheckResourceAttr(rName, "tags.key", "value"),
+					resource.TestCheckResourceAttr(rName, "tags.foo", "bar"),
+					resource.TestCheckResourceAttrPair(rName, "log_group_id", "huaweicloud_lts_group.test", "id"),
+					resource.TestCheckResourceAttrPair(rName, "log_stream_id", "huaweicloud_lts_stream.test", "id"),
+					resource.TestCheckResourceAttrSet(rName, "log_group_name"),
+					resource.TestCheckResourceAttrSet(rName, "log_stream_name"),
+				),
+			},
+			{
+				ResourceName:      rName,
+				ImportState:       true,
+				ImportStateVerify: true,
+				ImportStateIdFunc: testAccHostAccessConfigImportStateFunc(rName),
+			},
+		},
+	})
+}
+
+func testAccHostAccessConfigImportStateFunc(rName string) resource.ImportStateIdFunc {
+	return func(s *terraform.State) (string, error) {
+		rs, ok := s.RootModule().Resources[rName]
+		if !ok {
+			return "", fmt.Errorf("resource (%s) not found: %s", rName, rs)
+		}
+
+		name := rs.Primary.Attributes["name"]
+		if name == "" {
+			return "", fmt.Errorf("can not find the 'name' parameter in %s", rName)
+		}
+		return name, nil
+	}
+}
+
+func testHostAccessConfig_base(name string) string {
+	return fmt.Sprintf(`
+resource "huaweicloud_lts_group" "test" {
+  group_name  = "%[1]s"
+  ttl_in_days = 30
+}
+
+resource "huaweicloud_lts_stream" "test" {
+  group_id    = huaweicloud_lts_group.test.id
+  stream_name = "%[1]s"
+}
+`, name)
+}
+
+func testHostAccessConfig_basic(name string) string {
+	return fmt.Sprintf(`
+%[1]s
+
+# A host group without hosts
+resource "huaweicloud_lts_host_group" "test" {
+  name     = "%[2]s"
+  type     = "linux"
```

Comment:
Remove extra spaces and check the other parts.

Reply:
fixed